notebooks/rxrx3_core_benchmarks_openphenom.ipynb

```diff
@@ -0,0 +1,2858 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    assert embeddings_mrged[metadata_columns].shape[0] == embeddings_pcacs.shape[0]
</pre><p>Also minor, These few assert statements can be removed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/recursionpharma/EFAAR_benchmarking/pull/84/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>I prefer to keep them in case someone changes the aligner &amp; it causes a divide by zero</p>